lib/palindrome_permutation.rb

```diff
@@ -1,4 +1,17 @@
 
 def palindrome_permutation?(string)
-  raise NotImplementedError, "palindrome_permutation? not implemented"
-end
\ No newline at end of file
+  
+  letter_hash = {}
+  string.chars.map do |char|
+    letter_hash["#{char}"] = string.count(char)
```

Comment:
`string.count(char)` is an O(n) operation, so that makes this a total of O(n^2) operation.

It could be better doing this:
```suggestion
    letter_hash["#{char}"] = letter_hash["#{char}"].nil? ? 1 : letter_hash["#{char}"]  + 1
```